tests/lean/hashmap_on_disk/Main.lean

```diff
@@ -0,0 +1,10 @@
+import Base.Primitives
```

Comment:
This file is hand-written, correct?

Reply:
correct ... this one I'm leaving in here for the sake of unit-testing, until we have proper support for #assert -- right now, running the code does not check the #asserts yet

Reply:
As I understand that the `#assert` now works, I removed this file